core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -174,6 +174,17 @@ package object config {
   private[spark] val EVENT_LOG_CALLSITE_LONG_FORM =
     ConfigBuilder("spark.eventLog.longForm.enabled").booleanConf.createWithDefault(false)
 
+  private[spark] val EVENT_LOG_ENABLE_ROLLING =
+    ConfigBuilder("spark.eventLog.rollLog")
```

Comment:
I can see pros and cons regarding combining options: if we combine two options into one, we couldn't provide default max size when enabled, because we decided to turn this off by default (so default value should be 0 then). I'll leave options as they are, and if you feel OK to leave default value to be 0, I'll combine both.

Reply:
The default doesn't need to exist (that's why we have optional configs). But either is fine (one or two configs). I just tend to prefer less configs.